lib/AST/ASTScopeCreation.cpp

```diff
@@ -209,8 +209,8 @@ void ASTScope::
 }
 
 void ASTScope::expandFunctionBody(AbstractFunctionDecl *AFD) {
-  auto *const SF = AFD->getParentSourceFile();
-  SF->getScope().expandFunctionBodyImpl(AFD);
+  if (auto *const SF = AFD->getParentSourceFile())
```

Comment:
Unfortunately, that doesn't work because `getScope` is not a const member (and can't be because it lazily creates the scope). So, I'm going to leave this as-is.